example/setup.py

```diff
@@ -1,10 +1,26 @@
 #!/usr/bin/env python
 
 from setuptools import setup
+from setuptools.extension import Extension
 from distutils.command.build import build as _build
+import sys
+import os
+
+# if on windows platform, patch distutils lib.
+if sys.platform == "win32":
+    utils_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', 'winutil')
+    sys.path.append(utils_path)
+    from patchdistutils import runtime_patch
+    runtime_patch()
```

Comment:
This worries me a lot. It is required for every single Python package using cysignals to do this?

Reply:
Can you add an option to remove this? I was able to get cysignals compile with MSVC 2015 and Python 3.5 without this and a small change.

Reply:
You mean an option to not run the monkey patch while on a win32 platform ?